CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ When releasing, please add the new-release-boilerplate to docs/pallas/CHANGELOG.
 
 ## jax 0.4.33
 
+* Deprecations
+  * In {func}`jax.numpy.trim_zeros`, non-arraylike arguments or arraylike arguments
+    with `ndim != 1` are now deprecated, and in the future will result in an error.
+
```

Comment:
JAX v0.4.33 was just released, so we'll need to sync to main and then put this under jax 0.4.34

Reply:
Changelog is updated.